tests/unit/components/gallery/HDGallery.spec.js

```diff
@@ -1,33 +1,111 @@
 import { wrapperFactoryBuilder } from 'tests/unit/helpers';
 import HdGallery from '@/components/gallery/HdGallery.vue';
+import HdGalleryPlaceholder from '@/components/gallery/HdGalleryPlaceholder.vue';
 import ITEMS from '@/stories/mocks/GALLERY_ITEMS';
 
-const IMAGE_SELECTOR = '.gallery__image';
-
-const wrapperBuilder = wrapperFactoryBuilder(HdGallery, {
-  props: {
-    items: ITEMS,
-  },
-});
-
 describe('HdGallery', () => {
-  it('renders component', () => {
-    const wrapper = wrapperBuilder();
+  const build = (props) => {
+    const wrapper = wrapperFactoryBuilder(HdGallery, {
+      props: {
+        items: ITEMS,
+      },
+    })({ props });
```

Comment:
```suggestion
  const wrapperFactory = wrapperFactoryBuilder(HdGallery, {
      props: {
        items: ITEMS,
      },
    });
  const build = (...params) => {
    const wrapper = wrapperFactory(...params);
```

It could be something like that. You don't need to create the factory every time. You create it once and reuse as needed.

Reply:
Awesome suggestion, thanks =D

ce8a2c6

What do you think?